pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -686,6 +686,13 @@ let
       HOTPLUG_PCI_ACPI = yes; # PCI hotplug using ACPI
       HOTPLUG_PCI_PCIE = yes; # PCI-Expresscard hotplug support
 
+      # Enable memory hotplug support
```

Comment:
please add a bit more context; Eg: useful for VMware and enterprise hardware

Reply:
@zimbatm Sure thing. Please let me know if you find this new comment sufficient.